EventListener/CustomContentSubscriber.php

```diff
@@ -52,26 +52,35 @@ public static function getSubscribedEvents()
     {
         return [
             CoreEvents::VIEW_INJECT_CUSTOM_CONTENT => ['getContentInjection', 0],
+            CoreEvents::VIEW_INJECT_CUSTOM_ASSETS  => ['getAssetInjection', 0],
+           // CoreEvents::BUILD_MAUTIC_JS => ['getJSBuild', 0],
         ];
     }
 
+    /**
+     * @param CustomAssetsEvent $event
+     *
+     * @return CustomAssetsEvent
+     */
+    public function getAssetInjection(CustomAssetsEvent $event)
```

Comment:
Does the CustomAssetsEvent give us any indication of which page/section we are in so that we do not bloat all page loads?

Reply:
yeah, i am pretty sure, will double check shortly

Reply:
```
        $location = $this->router->getContext()->getPathInfo();

        if (preg_match('#campaigns/view/\d+$#', $location)) {
            $event->addScript('plugins/MauticContactLedgerBundle/Assets/js/datatables.min.js', 'bodyClose');
            $event->addStylesheet('plugins/MauticContactLedgerBundle/Assets/css/datatables.min.css');
        }

        return $event;
```
The $location variable gives us a page reference of sorts...